### PR TITLE
Extract related bandcamp items

### LINKED ISSUE #593: Display suggestions
When viewing an album or track on the web, Bandcamp offers suggestion under the heading "If you like …, you may also like". These should be parsed and displayed.

### PR DESCRIPTION
**Pull request by fynngodau**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [ ] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

Extracts recommended playlists as recommended playlists in NewPipeExtractor. Fixes #593.

| Recommended row on bandcamp website | Related items in NewPipe app |
|---|---|
| image | Screenshot_1618329234 |

### CODE REVIEW

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampStreamExtractor.java [hunk 1]
```diff
@@ -245,8 +246,17 @@ public StreamType getStreamType() {
     }
 
     @Override
-    public StreamInfoItemsCollector getRelatedItems() {
-        return null;
+    public PlaylistInfoItemsCollector getRelatedItems() {
+
+        PlaylistInfoItemsCollector collector = new PlaylistInfoItemsCollector(getServiceId());
+
+        Elements recommendedAlbums = document.getElementsByClass("recommended-album");
```
Comment (TobiGr) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampStreamExtractor.java:253]: `document` is null in `BandcampRadioStreamExtractor`. You need to override this method.

Comment (fynngodau) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampStreamExtractor.java:253]: Thanks. I should've run the bandcamp tests before opening the PR. Fixed through 36718767215d5a12becbf3176d5e010a461a338b.

**Review by TobiGr — APPROVED** [review]

**Comment by opusforlife2** [comment]
Am I missing something? In the RC I get "No videos" on any stream I try, including the one in the screenshot.

**Comment by fynngodau** [comment]
@opusforlife2 This change is not included in the RC for 0.21.2. I opened this PR six days ago, the RC PR https://github.com/TeamNewPipe/NewPipe/pull/6053 was posted 8 days ago.

**Comment by TobiGr** [comment]
Updated the RC

**Comment by opusforlife2** [comment]
Yeah, I tested RC2 and it works. 👍

**Comment by opusforlife2** [comment]
On a related note, maybe merged PRs shouldn't be added to the Release Issue and PR until the RC with those merged PRs is out? Merged PRs can still be added to the draft release notes on the Releases page to help keep track.

**Pull request merged**